src/postamble.js

```diff
@@ -189,6 +189,11 @@ function callMain(args) {
     var start = Date.now();
 #endif
 
+#if ABORT_ON_WASM_EXCEPTIONS
+    // See abortWrapperDepth in preamble.js!
+    abortWrapperDepth += 2; 
```

Comment:
Dos this no work with `+= 1`?

Reply:
No, because the check in `instrumentWasmExportsWithAbort` is `abortWrapperDepth > 1` - i.e. it will rethrow anything that's not at the top level until it hits top level. So the += 2 basically makes `main` behave as if it's never top level and all the exceptions are rethrown to be handled in `callMain`